test/instrumentation/modules/tedious.js

```diff
@@ -18,22 +18,28 @@ const test = require('tape')
 const mockClient = require('../../_mock_http_client')
 const version = require('tedious/package').version
 
-let connection
+let connOpts
 const hostname = process.env.MSSQL_HOST || '127.0.0.1'
 
 if (semver.gte(version, '4.0.0')) {
-  connection = {
+  connOpts = {
     server: hostname,
     authentication: {
       type: 'default',
       options: {
         userName: 'SA',
         password: process.env.SA_PASSWORD || 'Very(!)Secure'
       }
+    },
+    options: {
+      // Tedious@9 changed to `trustServerCertificate: false` by default.
+      trustServerCertificate: true,
+      // Silence deprecation warning in tedious@8.
+      validateBulkLoadParameters: true
     }
```

Comment:
I see we're adding some extra configuration options here in order to ensure the tests run clean.